x-pack/plugins/apm/public/components/alerting/register_apm_alerts.ts

```diff
@@ -16,11 +16,11 @@ import {
 import type { ObservabilityRuleTypeRegistry } from '../../../../observability/public';
 import { ENVIRONMENT_ALL } from '../../../common/environment_filter_values';
 import { AlertType } from '../../../common/alert_types';
-
-// copied from elasticsearch_fieldnames.ts to limit page load bundle size
```

Comment:
@spong does this change have an effect on the bundle size? It looks like there was a small net increase on your PR, but @dgieselaar added this recently to limit the size.

Reply:
Yeah, looks to increase the bundle by 4.9KB -- @madirey was there an explicit reason for pulling these back in or just a result from working through all the conflicts?


id | before | after | diff
-- | -- | -- | --
apm | 32.2KB | 37.1KB | +4.9KB

Reply:
Sorry, just saw this, checking...

Reply:
@spong Yeah, looks like maybe I just took the wrong change on the rebase...